packages/cardano-graphql/codegen.yml

```diff
@@ -10,4 +10,10 @@ config:
   strictScalars: true
   scalars:
     DateTime: string
-    Int64: number
+    # Review: it actually seems to be serialized to number in response json
+    # Suggested solution:
+    # Fork (and possibly open a PR) this dependency:
+    # https://github.com/prisma-labs/graphql-request/blob/master/src/index.ts#L440
+    # Either patching, or adding an option to provide a custom JSON serializer,
```

Comment:
We hit a similar challenge recently with the JSON parser used in the Ogmios client. Ended up forking since the original product didn't appear to have good support anyway. https://github.com/CardanoSolutions/json-schema-to-typescript. I do think the custom parser is the best design here, and one that would have potential to be accepted upstream